js/ad_manager_vast.js

```diff
@@ -451,7 +447,7 @@ OO.Ads.manager(() => {
      * @param {object} vpaidAd The vpaidAd object.
      * @param {string} funcName The function name.
      * @param {array} params The array of params.
-     * @returns {null|function} Returns function with vpaidAd and params arguments.
+     * @returns {null|function} function with vpaidAd and params arguments.
```

Comment:
It doesn't return function, just executes and return the result. Please fix to `*`